bin/regen-protos.sh

```diff
@@ -10,7 +10,9 @@ echo "prebuilt binaries for your computer into nanopb-0.4.7"
 cd protobufs
 ../nanopb-0.4.7/generator-bin/protoc --nanopb_out=-v:../src/mesh/generated/ -I=../protobufs meshtastic/*.proto --experimental_allow_proto3_optional
 
-# cd ../src/mesh/generated/meshtastic
+cd ../src/mesh/generated/meshtastic
+rename .c .cpp *.c
```

Comment:
I think it would be better to use the nanopb option to change the default extension to .cpp:
```
  -S EXTENSION, --source-extension=EXTENSION
                        Set extension to use for generated source files.
                        [default: .c]
```

Reply:
Also valid concern by trunk : `Use ./*glob* or -- *glob* so names with dashes won't become options.  shellcheck/SC2035` - better ditch that line and go with the other suggestion

Reply:
This isn't an option in protoc, that I see. 
```
jbennett@Celeborn:~/src/firmware$ ./nanopb-0.4.7/generator-bin/protoc --source-extension=.cpp
Unknown flag: --source-extension
```

Reply:
That is not straightforward. But got it.